librubyfmt/src/ruby.rs

```diff
@@ -93,12 +94,10 @@ extern "C" {
 }
 
 pub fn current_exception_as_rust_string() -> String {
-    unsafe {
-        let res = eval_str("$!.inspect").expect("this can't fail");
-        let ptr = rubyfmt_rstring_ptr(res);
-        let length = rubyfmt_rstring_len(res);
-        String::from_raw_parts(ptr as _, length as _, length as _)
```

Comment:
This was the double free mentioned in the PR description. Clearly this branch isn't executed on any files we currently test on, but this would result in the same segfault seen earlier